modules/special-rules/nav.adoc

```diff
@@ -18,43 +18,59 @@
   *** xref:vortex-missile.adoc[]
 
  ** xref:special-abilities.adoc[]
-  *** xref:anti-personnel-ability.adoc[]
-  *** xref:artillery.adoc[]
-  *** xref:assault.adoc[]
-  *** xref:bridge.adoc[]
-  *** xref:brood-telepathy.adoc[]
-  *** xref:cavalry.adoc[]
-  *** xref:close-support.adoc[]
-  *** xref:deathwing.adoc[]
-  *** xref:engineers.adoc[]
-  *** xref:fast.adoc[]
-  *** xref:flak.adoc[]
-  *** xref:heavy-weapons.adoc[]
-  *** xref:hero.adoc[]
-  *** xref:hit-and-run.adoc[]
-  *** xref:infiltrators.adoc[]
-  *** xref:jump-packs.adoc[]
-  *** xref:kroot.adoc[]
-  *** xref:long-range.adoc[]
-  *** xref:medic.adoc[]
-  *** xref:open-top.adoc[]
-  *** xref:ordo-malleus.adoc[]
-  *** xref:psyker.adoc[]
-  *** xref:rampage.adoc[]
-  *** xref:rapid-fire.adoc[]
-  *** xref:ravenwing.adoc[]
-  *** xref:recovery.adoc[]
-  *** xref:save.adoc[]
-  *** xref:self-propelled-artillery.adoc[]
-  *** xref:shootier.adoc[]
-  *** xref:siege.adoc[]
-  *** xref:skimmer.adoc[]
-  *** xref:space-wolves.adoc[]
-  *** xref:stubborn.adoc[]
-  *** xref:supreme-commander.adoc[]
-  *** xref:tank-destroyer.adoc[]
-  *** xref:transport-x.adoc[]
-  *** xref:tunneller.adoc[]
+  *** xref:data-sheet-modifiers.adoc[]
```

Comment:
Note that I have tidied up the nav, and I made the headings into actual topics, like the rest of the project. It is expedient not to bother while we're roughing it out, but now is a good time, plus I wanted to move the descriptions and examples from special-abilities.adoc to the new topics.